### DISCUSSION TITLE
<ele> tag value not showing in Osmand

### DISCUSSION THREAD

**Discussion by gbrown2036** [osmandapp/OsmAnd#19977] [Q&A]
Below is a sample waypoint in my GPX file but the Osmand UI doesn't show the elevation value of -1712 which is in my `<ele>` tag.  What am I doing wrong?  Thanks

```
  <wpt xmlns:osmand="https://osmand.net" lat="84.177317" lon="-84.177317">
  <ele>-1712</ele>
  <time>2024-05-15T21:14:09Z</time>
  <name>~GREATSTRUCTURE</name>
  <cmt/>
  <sym>Pin, Green</sym>
  <type>user</type>
  <extensions xmlns:osmand="https://osmand.net">
	<osmand:icon>internet_access</osmand:icon>
	<osmand:background>circle</osmand:background>
	<osmand:color>#8fce00</osmand:color>
  </extensions>
  </wpt>
```

**Comment by vshcherb** [comment]
You have elevation -1712 m under sea level, so they are not visible... If you have 3D track enabled

**Reply by gbrown2036** [reply]
Thanks for responding.  I changed it to a **positive** 1712 and it still doesn't show.  I don't believe I have 3D track enabled.

Is there a ticket anywhere related to any changes made in the past few years?  It didn't used to do this with negative values in the past.

(As a workaround, I can add the value to the `<cmt>` tag but I'd prefer to be able to use the `<ele>` tag)

Thanks!

**Reply by gbrown2036** [reply]
In looking at the sample "Favorites in GPX file" section of this [link](https://osmand.net/docs/user/personal/favorites) I don't even see the `<ele>` tag listed or anywhere else to put an elevation.  

Am I missing something?

**Comment by dmpr0** [comment]
I have tried to add `ele` tag manually, data from tag shows in waypoint details.

<img src="https://github.com/osmandapp/OsmAnd/assets/9365308/e30c875e-540d-42b4-a2c0-c0db57747016"  width="300" height="auto"/>

**Reply by gbrown2036** [reply]
Hmmm.  I just tried with the below and I can only see the `<cmt> `value and not the `<ele>` value.  Are you using Android?

```
  <wpt xmlns:osmand="https://osmand.net" lat="84.177317" lon="-84.177317">
  <ele>1712</ele>
  <time>2024-05-15T21:14:09Z</time>
  <name>~GREATSTRUCTURE</name>
  <cmt>-1712 foo</cmt>
  <sym>Pin, Green</sym>
  <type>user</type>
  <extensions xmlns:osmand="https://osmand.net">
	<osmand:icon>internet_access</osmand:icon>
	<osmand:background>circle</osmand:background>
	<osmand:color>#8fce00</osmand:color>
  </extensions>
  </wpt>
```
